package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "instagram-download",
   "description": "Download all the Instagram JSON data and media for a user.",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
@rtorr Sorry, one more thing, can you revert this? I normally use `npm version patch` to have separate commits for version bumps.